lib/LightPanels/LightPanels.js

```diff
@@ -38,63 +35,8 @@ module.exports = kind(
 	/**
 	* @private
 	*/
-	defaultKind: LightPanel,
-
-	/**
-	* @private
-	*/
-	handlers: {
-		onSpotlightContainerEnter: 'enter',
-		onSpotlightContainerLeave: 'leave'
-	},
-
-	/**
-	* @private
-	*/
-	animateTo: function (index) {
-		this.updateSpottability(this.index, index);
-		LightPanels.prototype.animateTo.apply(this, arguments);
-	},
-
-	/**
-	* @private
-	*/
-	indexChanged: function (was, is) {
-		this.updateSpottability(was, is);
-		LightPanels.prototype.indexChanged.apply(this, arguments);
-	},
-
-	/**
-	* @private
-	*/
-	updateSpottability: function (from, to) {
-		var panels = this.getPanels(),
-			panelPrev = panels[from],
-			panelNext = panels[to];
-
-		if (panelPrev) {
-			panelPrev.spotlightDisabled = true;
-			if (this._hasSpotlightFocus) {
-				Spotlight.unspot();
-			}
-		}
-		if (panelNext) {
-			panelNext.spotlightDisabled = false;
-		}
-	},
-
-	/**
-	* @private
-	*/
-	enter: function () {
-		this._hasSpotlightFocus = true;
-	},
-
-	/**
-	* @private
-	*/
-	leave: function () {
-		this._hasSpotlightFocus = false;
-	}
+	defaultKind: LightPanel
```

Comment:
More optional ... I realize it's not needed right now but should we make it a practice that all kinds have at least one class (e.g. `moon-lightpanels') for ease in customization?